wherescape/connectors/hubspot/collect_data.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from datetime import datetime
+from ...wherescape import WhereScape
+from .process_data import hubspot_process_results
+
+""" 
+this module retrieves the data from Wherescape
+"""
+
+
+def hubspot_load_data():
+    """
+    This method collects all the data from a table and sends it to be processed.
+    """
+    start_time = datetime.now()
+    logging.info("connecting to WhereScape")
+    wherescape_instance = WhereScape()
+    logging.info(
+        "Start time: %s for hubspot_load_data"
+        % start_time.strftime("%Y-%m-%d %H:%M:%S")
+    )
+    logging.info("post load")
+    table_name = f"{wherescape_instance.schema}.{wherescape_instance.table}"
+    sql = f"select * from {table_name}"
+
+    result = wherescape_instance.query_target(sql)
+    access_token = hubspot_get_token(wherescape_instance, table_name)
+    column_names = wherescape_instance.get_columns()[0]
+
+    if len(result) > 0:
+        hubspot_process_results(access_token, result, column_names, table_name)
```

Comment:
I think another abstraction level / module is handy. since the method `hubspot_load_data` now does more that loading the data. See also line 7